static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -127,6 +131,7 @@ export const shouldUseOzoneAdaptor = (): boolean =>
 
 export const shouldIncludeAppNexus = (): boolean =>
     isInAuRegion() ||
+    (isInUsRegion() && isInAppnexusUSAdapterTestVariant()) ||
     ((config.get('switches.prebidAppnexusUkRow') && !isInUsRegion()) ||
```

Comment:
Is this `isInUsRegion` redundant now?

Reply:
do you mean this condition?  
`config.get('switches.prebidAppnexusUkRow') & !isInUsRegion()` ?

I tried to remove it but then it shows in US when the variant is false. I wonder, do we still need the switch `switches.prebidAppnexusUkRow` ? If not, i can remove it altogether since is now going to be applicable in all regions?

Reply:
Yep, that's the condition I meant.

I was just thinking that too; do we need any of the region checks at all? The whole adapter is now regionalised so it doesn't matter which region it gets served in.

It all depends on if the test is regionalised too; if it is then you only need to test the variant and the switch (as far as I can see!)

Reply:
Yap, agree. let me simplify it.